recipes/aws-crt-cpp/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import CMake, ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class AwsCrtCpp(ConanFile):
+    name = "aws-crt-cpp"
+    description = "C++ wrapper around the aws-c-* libraries. Provides Cross-Platform Transport Protocols and SSL/TLS implementations for C++."
+    topics = ("conan", "aws")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/awslabs/aws-crt-cpp"
+    license = "Apache-2.0",
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("aws-c-event-stream/0.2.7")
+        self.requires("aws-c-common/0.6.7")
+        self.requires("aws-c-io/0.10.5")
+        self.requires("aws-c-http/0.6.5")
+        self.requires("aws-c-auth/0.6.0")
+        self.requires("aws-c-mqtt/0.7.6")
+        self.requires("aws-c-s3/0.1.19")
+        self.requires("aws-checksums/0.1.11")
+
```

Comment:
Does it need a `validate` method?
It's a c++ method, so it maybe has a minimum c++ version?

Reply:
It's C++11 which is the minimum CCI requires 🤷